src/schema/v2/users.ts

```diff
@@ -12,17 +12,36 @@ export const Users: GraphQLFieldConfig<void, ResolverContext> = {
     ids: {
       type: new GraphQLList(GraphQLString),
     },
+    term: {
+      type: GraphQLString,
+      description:
+        "If present, will search by term, cannot be combined with `ids`",
+    },
   }),
-  resolve: async (_root, args, { usersLoader }) => {
-    if (!usersLoader) return null
+  resolve: async (_root, args, { usersLoader, matchUsersLoader }) => {
+    const { ids, term } = args
+
+    if (ids && term) throw new Error("Must provide either `ids` or `term`")
+
+    if (!usersLoader || !matchUsersLoader)
+      throw new Error(
+        "You need to pass a X-Access-Token header to perform this action"
+      )
+
     const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
 
-    const { body, headers } = await usersLoader({
-      id: args.ids,
-      total_count: true,
```

Comment:
Was the total count header just not set previously since it looks like this is only supported after your PR https://github.com/artsy/gravity/pull/15704/files?

Reply:
The Gravity update was needed for the match loader to be able to have the total count, this `usersLoader` that was being used was fine.